examples/knative/powercli/kn-pcli-tag/handler.ps1

```diff
@@ -0,0 +1,82 @@
+Function Process-Init {
+   Write-Host "$(Get-Date) - Processing Init`n"
+
+   if(${env:TAG_SECRET}) {
+      $jsonSecrets = ${env:TAG_SECRET} | ConvertFrom-Json
+   } else {
+      Write-Host "$(Get-Date) - K8s secrets `$env:TAG does not look to be defined"
+      break
```

Comment:
Break or throw?

Reply:
Break, because we don't want to continue if we're not able to get secret